control/bdalg.py

```diff
@@ -280,7 +281,10 @@ def append(*sys):
     >>> sys = append(sys1, sys2)
 
     """
-    s1 = sys[0]
+    if not isinstance(sys[0], StateSpace):
+        s1 = ss._convert_to_statespace(sys[0])
```

Comment:
FYI, this function already checks if the argument is a `StateSpace` object and just returns if so => you can replace lines 284 to 287 with `s1 = ss._convert_to_statespace(sys[0])`.

Reply:
good point. fixed in next commit.